src/assets/svgs/icon-close.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
Thought: Do we benefit from having `role="img"` or `aria-hidden="true"` on non-meaningful inline SVGs, to ensure they're not announced to assistive tech? [Ref](https://css-tricks.com/accessible-svgs/)

Reply:
I've omitted `<title>` from not meaningful SVGs, would that be enough?

Reply:
Added `focusable="false"` and `aria-hidden="true"`.